src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/PreferStreamReadAsyncMemoryOverloadsTests.cs

```diff
@@ -19,10 +20,10 @@ class C
 {
     public void M()
     {
-        using (FileStream fs = File.Open(""file.txt"", FileMode.Open))
+        using (FileStream s = File.Open(""file.txt"", FileMode.Open))
```

Comment:
Renamed the stream object to `s` in all the unit tests so that I could format the source code text the same way for all tests.